manifests/params.pp

```diff
@@ -255,6 +255,7 @@
   $file_default_mode = '0640'
 
   $loglevel = 'INFO'
+  $rd_log4j_rotatedaily = true
```

Comment:
since this is a static variable, can you just drop the line here and set the variable in init.pp to true?